sklearn/linear_model/logistic.py

```diff
@@ -914,15 +949,21 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
         as ``n_samples / (n_classes * np.bincount(y))``
 
     max_iter : int
-        Useful only for the newton-cg and lbfgs solvers. Maximum number of
-        iterations taken for the solvers to converge.
+        Useful only for the newton-cg, sag and lbfgs solvers.
+        Maximum number of iterations taken for the solvers to converge.
 
     random_state : int seed, RandomState instance, or None (default)
         The seed of the pseudo random number generator to use when
         shuffling the data.
 
-    solver : {'newton-cg', 'lbfgs', 'liblinear'}
+    solver : {'newton-cg', 'lbfgs', 'liblinear', 'sag'}
         Algorithm to use in the optimization problem.
+        - For small datasets, 'liblinear' is a good choice, whereas 'sag' is
+            faster for large ones.
+        - For multiclass problems, only 'newton-cg' and 'lbfgs' handle
+            multinomial loss; 'sag' and 'liblinear' are limited to
+            one-versus-rest schemes.
+        - 'newton-cg', 'lbfgs' and 'sag' only handle L2 penalty.
```

Comment:
I would add a note to state that `'sag'` fast convergence is only guaranteed on features with approximately the same scale, possibly by preprocessing the data with a scaler from `sklearn,preprocessing`.

A similar remark would also hold for the docstring of the `Ridge` class.